### DISCUSSION TITLE
Mobile album list view and album search box

### DISCUSSION THREAD

**Discussion by adambomb1011** [immich-app/immich#8240] [Feature Request]
### The feature

The web version contains a list view for albums, and also the ability to search by album name. Can this be translated also to mobile?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by frederikb96** [comment]
I am also interested in this :)
The ability to search for album name is also asked for in #8170 :)

**Comment by bo0tzz** [comment]
#2407

**Reply by alex-sherwin** [reply]
#2407 doesn't cover list view in mobile app
